types/prismjs/index.d.ts

```diff
@@ -1,385 +1,395 @@
-    selector?: string | undefined;
-    element?: Element | undefined;
-    language?: string | undefined;
-    grammar?: Grammar | undefined;
-    code?: string | undefined;
-    highlightedCode?: string | undefined;
-    type?: string | undefined;
-    content?: string | undefined;
-    tag?: string | undefined;
-    classes?: string[] | undefined;
-    attributes?: Record<string, string> | undefined;
-    parent?: Array<string | Token> | undefined;
-}
-
-export namespace util {
-    interface Identifier {
-        value: number;
-    }
-
-    /** Encode raw strings in tokens in preparation to display as HTML */
-    function encode(tokens: TokenStream): TokenStream;
-
-    /** Determine the type of the object */
-    function type(o: null): 'Null';
-    function type(o: undefined): 'Undefined';
-    // tslint:disable:ban-types
-    function type(o: boolean | Boolean): 'Boolean';
-    function type(o: number | Number): 'Number';
-    function type(o: string | String): 'String';
-    function type(o: Function): 'Function';
-    // tslint:enable:ban-types
-    function type(o: RegExp): 'RegExp';
-    function type(o: any[]): 'Array';
-    function type(o: any): string;
-
-    /** Get the unique id of this object or give it one if it does not have one */
-    function objId(obj: any): Identifier;
-
-    /** Deep clone a language definition (e.g. to extend it) */
-    function clone<T>(o: T): T;
-}
-
-export type GrammarValue = RegExp | TokenObject | Array<RegExp | TokenObject>;
-export type Grammar = GrammarRest | Record<string, GrammarValue>;
-export interface GrammarRest {
-    keyword?: GrammarValue | undefined;
-    number?: GrammarValue | undefined;
-    function?: GrammarValue | undefined;
-    string?: GrammarValue | undefined;
-    boolean?: GrammarValue | undefined;
-    operator?: GrammarValue | undefined;
-    punctuation?: GrammarValue | undefined;
-    atrule?: GrammarValue | undefined;
-    url?: GrammarValue | undefined;
-    selector?: GrammarValue | undefined;
-    property?: GrammarValue | undefined;
-    important?: GrammarValue | undefined;
-    style?: GrammarValue | undefined;
-    comment?: GrammarValue | undefined;
-    'class-name'?: GrammarValue | undefined;
-
-    /**
-     * An optional grammar object that will appended to this grammar.
-     */
-    rest?: Grammar | undefined;
-}
 
-/**
- * The expansion of a simple `RegExp` literal to support additional properties.
- */
-export interface TokenObject {
-    /**
-     * The regular expression of the token.
-     */
-    pattern: RegExp;
-
-    /**
-     * If `true`, then the first capturing group of `pattern` will (effectively) behave as a lookbehind
-     * group meaning that the captured text will not be part of the matched text of the new token.
-     */
-    lookbehind?: boolean | undefined;
+declare namespace Prism {
+    const languages: Languages;
+    const plugins: Record<string, any>;
+    let disableWorkerMessageHandler: boolean;
```

Comment:
Both `disableWorkerMessageHandler` and `manual` start out as undefined, so the type should be `boolean | undefined`.

Reply:
Are you sure? The default value for both variables is `false`.
https://github.com/PrismJS/prism/blob/master/prism.js#L51-L78

Reply:
![image](https://user-images.githubusercontent.com/801549/151045137-4c60757c-df91-4110-9feb-3ab882dec763.png)